girder/models/setting.py

```diff
@@ -209,6 +210,18 @@ def validateCoreUploadMinimumChunkSize(self, doc):
             'Upload minimum chunk size must be an integer >= 0.',
             'value')
 
+    def validateCoreFilesystemDefaultFileMode(self, doc):
+        try:
+            doc['value'] = int(doc['value'])
```

Comment:
Shouldn't this be `int(doc['value'], 8)`, since most users would probably expect this to be in octal notation?